_posts/2024-02-29-defna-board-member-recruitment.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Join DEFNA! Board Member Recruitment"
+date: 2024-02-29 12:30:00
+categories: ""
+author_name : DEFNA
+author_url : /about
+author_avatar: defna
+show_avatar : true
+read_time : ""
+feature_image: 2024-02-29
+show_related_posts: false
+permalink: /announcements/2024/02/29/defna-board-member-recruitment/
+---
+
+
+Django Events Foundation North America (DEFNA) is looking for another board member. We have a nine-member board; one of our board members is stepping down as soon as a replacement is confirmmed. We are looking for a new board member who is interested in growing the DjangoCon US community.
+
```

Comment:
```suggestion

Currently we don't have a Corporate Treasurer to manage our finances. We're considering hiring an accounting firm to do most of the finance work. This means the new Treasurer would focus more on overseeing our finances instead of doing all the data entry. If you're passionate about DjangoCon US and have some experience or interest in managing finances, we'd love to hear from you.

```